rust/protocol/src/curve/curve25519.rs

```diff
@@ -140,10 +173,13 @@ impl PrivateKey {
         bool::from(cap_r_check.as_bytes().ct_eq(&cap_r))
     }
 
+    /// Return the bytes of a [`PublicKey`] that another party can use to validate against
+    /// [`Self::verify_signature`].
     pub fn derive_public_key_bytes(&self) -> [u8; PUBLIC_KEY_LENGTH] {
```

Comment:
That's not the only use, it's also for key agreement! I'd drop the use part completely again.